### DISCUSSION TITLE
Force FASTER to always use disk for persistence

### DISCUSSION THREAD

**Discussion by ironhide91** [microsoft/FASTER#754] [Q&A] [answered]
How to force FASTER to always use disk for persistence no in-memory ?

The reason for asking this is, I don't see to log file create for both log and object log.
The size of the cache values are on average 150 MB.
I don't want this to be in my process memory.

**Comment by badrishc** [comment, marked as answer]
Make the page size and memory size small, then FASTER will flush to disk when you run out of memory.
